native/src/tests/oberon.rs

```diff
@@ -0,0 +1,172 @@
+use crate::{didcomm::Error, proto::{security::*}};
+
+
+#[test]
+fn test_create_token() {
+    let req = CreateOberonTokenRequest {
+        sk: vec![1,2,3],
+        data: vec![1,2,3],
+        blinding: Vec::new(),
+    };
+
+
+    let rep = crate::Oberon::token(&req);
+    assert!(matches!(rep.unwrap_err(), Error::InvalidField("invalid secret key provided")));
+    
+    let req = CreateOberonTokenRequest {
+        sk: vec![
+            16, 133, 126, 11, 192, 153, 22, 14, 53, 214, 99, 40, 66, 194, 96, 30, 19, 86, 137, 107,
+            150, 49, 104, 202, 209, 80, 128, 182, 15, 154, 34, 57, 100, 51, 175, 108, 12, 56, 6,
+            76, 46, 173, 247, 255, 184, 165, 228, 127, 145, 65, 171, 195, 44, 164, 3, 16, 132, 43,
+            108, 82, 63, 136, 116, 3, 93, 1, 226, 152, 197, 152, 61, 212, 185, 32, 195, 211, 37,
+            206, 242, 31, 72, 79, 83, 71, 197, 102, 202, 129, 95, 19, 105, 34, 22, 46, 124, 94
+        ],
+        data: Vec::new(),
+        blinding: Vec::new(),
+    };
```

Comment:
Could we use this as an example .NET test?

Reply:
Yes, but I'll add tests in the net project that use didkey instead.